task/init/0.2/init.yaml

```diff
@@ -41,7 +41,7 @@ spec:
         echo "Determine if Image Already Exists"
         # Build the image when rebuild is set to true or image does not exist
         # The image check comes last to avoid unnecessary, slow API calls
-        if [ "$REBUILD" == "true" ] || [ "$SKIP_CHECKS" == "false" ] || ! oc image info $IMAGE_URL &>/dev/null; then
+        if [ "$REBUILD" == "true" ] || [ "$SKIP_CHECKS" == "false" ] || ! skopeo inspect docker://$IMAGE_URL &>/dev/null; then
```

Comment:
Consider using `--raw` or at least `--no-tags`, the default inspect is very slow (raw should be fastest I think)

Reply:
done